tests/csm/rest/test_csm_load.py

```diff
@@ -70,21 +75,56 @@ def setup_class(cls):
         assert s3acc_already_present
         cls.log.info("[Completed]: Setup class")
         cls.default_cpu_usage = False
+        cls.buckets_created = []
         cls.iam_users_created = []
+        cls.ha_obj = HAK8s()
+        cls.failed_pod = []
+        cls.restore_pod = cls.deployment_backup = cls.deployment_name = cls.restore_method = None
+        cls.num_replica = 1
```

Comment:
will this always be 1? or default settings are for replicaset?

Reply:
removed